06-building-a-program-start-to-finish/writing-tests/output/invertedSpec.js

```diff
@@ -0,0 +1,152 @@
+const validBook = require('./test/testFiles/validBook.json');  //  book with valid contents
+const invalidBook = require('./test/testFiles/invalidBook.json'); //  book with invalid content
+const emptyBook = require('./test/testFiles/emptyBook.json');   //  empty book
+const malformedJSON = require('./test/testFiles/malformed.json'); // empty book
+const smallBook = require('./test/testFiles/smallValidBook.json'); // book with few words
+const InvertedIndex = require('../src/js/inverted.js').InvertedIndex; // Inverted Index class file
+
```

Comment:
A base filename should exactly match the name of its default export. Your filenames and variable names should be in camelCase. It aids readability.